commands/bridge_configure.go

```diff
@@ -21,26 +33,31 @@ func runBridgeConfigure(cmd *cobra.Command, args []string) error {
 	defer backend.Close()
 	interrupt.RegisterCleaner(backend.Close)
 
-	target, err := promptTarget()
-	if err != nil {
-		return err
+	if bridgeConfigureTarget == "" {
+		bridgeConfigureTarget, err = promptTarget()
```

Comment:
not your code, but I just found out that there is an annoying thing here: each bridge implementation are registered with a `init()` function and the order is not deterministic :(
One run will have the github one first, another will have the launchpad one first.
We need a way to make that consistent.

Reply:
this can be dealt with in another PR btw

Reply:
Agreed having a different order every time we want to configure is a little weird. Will be dealing with this ASAP ;)